src/dbnode/storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -275,6 +323,7 @@ func (s *commitLogSource) ReadData(
 	}
 
 	// Read / M3TSZ encode all the datapoints in the commit log that we need to read.
+	s.metrics.data.readingCommitlogs.Update(1)
```

Comment:
could you do same thing we normally do - use an atomic and have a background routine emitting the guage regularly. that way you'd get more than a single datapoint indicating what's going on.